include/constants/game_stats.h

```diff
@@ -19,8 +19,16 @@
 #define GAME_STAT_BADGE_GET           22
 #define GAME_STAT_UNK26               26
 #define GAME_STAT_UNK30               30
+#define GAME_STAT_UNK42               42
 #define GAME_STAT_UNK46               46
 #define GAME_STAT_BATTLE_POINTS       69
 #define GAME_STAT_UNK70               70
+#define GAME_STAT_UNK77               77
```

Comment:
I'm gonna be real I started trying to document game stats, realized we can document almost all of them since we have the gmm, and decided I'm just gonna document them when I get to that actual file before this balloons